sdk-setup/src/sdk-chroot

```diff
@@ -294,6 +294,10 @@ prepare_user() {
 
     echo "$user ALL=NOPASSWD: ALL" > ${sdkroot}/etc/sudoers.d/$user
     chmod 0440 ${sdkroot}/etc/sudoers.d/$user
+    
+    if [[ $user == *['!'.]* ]] ; then
```

Comment:
What's the '!' there for? Wouldn't just `*[.]*` suffice?

Reply:
Yes, it seems so